content/300-guides/200-deployment/110-deployment-guides/600-deploying-migrations-from-a-local-environment.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Deploying migrations from a local environment'
+metaTitle: 'Deploying migrations from a local environment'
+metaDescription: 'Learn how to deploy Node.js and TypeScript applications that are using Prisma Client locally.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+There are instances whereby you might want to run a deployment migration locally. This page outlines some examples of how you can do that and **why its generally a bad practice**.
```

Comment:
* Would say 'not recommended practice' instead of bad
* Can we expand on 'might want to run a deployment migration locally' - maybe something like 'There are **two** scenarios where you might consider deploying migrations directly from a local environment to a production environment' and then list the two use cases: you only have a local CI / CD pipeline + you are baselining a production environment